[2023-06-15T07:22:43.281+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [queued]>
[2023-06-15T07:22:43.317+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [queued]>
[2023-06-15T07:22:43.318+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T07:22:43.320+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-06-15T07:22:43.321+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T07:22:43.357+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2023-06-15 07:20:10.524334+00:00
[2023-06-15T07:22:43.367+0000] {standard_task_runner.py:55} INFO - Started process 5723 to run task
[2023-06-15T07:22:43.371+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'format_to_parquet_task', 'manual__2023-06-15T07:20:10.524334+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpu7n48ng9']
[2023-06-15T07:22:43.376+0000] {standard_task_runner.py:83} INFO - Job 325: Subtask format_to_parquet_task
[2023-06-15T07:22:43.485+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.format_to_parquet_task manual__2023-06-15T07:20:10.524334+00:00 [running]> on host 658b05ff6f1a
[2023-06-15T07:22:43.606+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T07:20:10.524334+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-15T07:20:10.524334+00:00
[2023-06-15T07:22:43.614+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_dag.py", line 25, in format_to_parquet
    raise ValueError("No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.")
ValueError: No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.
[2023-06-15T07:22:43.651+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=data_ingestion_gcs_dag, task_id=format_to_parquet_task, execution_date=20230615T072010, start_date=20230615T072243, end_date=20230615T072243
[2023-06-15T07:22:43.679+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 325 for task format_to_parquet_task (No CSV files found. Make sure the dataset has been correctly downloaded and unzipped.; 5723)
[2023-06-15T07:22:43.715+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-06-15T07:22:43.749+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
